nvflare/edge/widgets/evaluator.py

```diff
@@ -65,9 +69,13 @@ def __init__(
         self.model = None
         self.data_loader = None
         self.tb_writer = None
+        self.max_workers = max_workers
```

Comment:
Should validate to make sure max_workers is a positive int.
Similarly eval_frequency should be validated too.
You can use nvflare.fuel.utils.validation_utils to do these. It has many useful funcs like check_positive_int, check_positive_number, and many others.

Reply:
added